app/models/profile.rb

```diff
@@ -122,6 +124,22 @@ def has_proofed_before?
 
   private
 
+  def track_for_attempts_api(status)
+    if initiating_service_provider&.irs_attempts_api_enabled?
+      tracker = IrsAttemptsApi::Tracker.new(
+        session_id: nil,
+        request: nil,
+        user: user,
+        sp: initiating_service_provider,
+        cookie_device_uuid: nil,
+        sp_request_uri: nil,
+        enabled_for_session: true,
+        analytics: nil,
+      )
+      tracker.fraud_review_adjudicated(determination: status.to_s)
+    end
+  end
```

Comment:
another idea might be to the tracker and just have the call sites call `fraud_review_adjudicated` directly. This has the bonus of being able to see what directly is passed in to `determiniation` and also we can remove the `expect_any_instance_of(IrsAttemptsApi::Tracker).to receive` in specs because we can do `expect(profile.irs_attemts_api).to receive`

```suggestion
  def irs_attempts_api_tracker
    return @irs_attempts_api_tracker if defined?(@irs_attempts_api_tracker)
    @irs_attempts_api_tracker = if initiating_service_provider&.irs_attempts_api_enabled?
      IrsAttemptsApi::Tracker.new(
        session_id: nil,
        request: nil,
        user: user,
        sp: initiating_service_provider,
        cookie_device_uuid: nil,
        sp_request_uri: nil,
        enabled_for_session: true,
        analytics: nil,
      )
    end
  end
```